### ISSUE TITLE
Ship name and ID reverts to entirely lower case in EDDI

### ISSUE THREAD

**Issue by RSMLancaster** [cmdrmcdonald/EliteDangerousDataProvider#200]
Hi jgm,

I have noticed a minor issue with EDDI version 2.3.0-b2 (same with b1) in that the imported ship name and ID which appear in Capitals in ED, revert to entirely lower case when EDDI imports them.

Same issue when exporting from EDDI to Coriolis.

Only a minor point, but just letting you know.

**Comment by cmdrmcdonald** [comment]
Elite and EDDI are both case-sensitive, however the font that displays the ship names in-game is an all-caps font so this isn't obvious.

**Comment by RSMLancaster** [comment]
Interesting, I just went to Livery in ED and used Caps lock for ship name which EDDI imported as Caps, but when I did the same for ship ID EDDI imports the ship ID in Caps and then reverts ship name in lower case!

tried this a couple of times and it appears that either the ships name can be in capitals or the iD, but apparently not both. Very strange.....

Probably something at the ED end I guess, not that important.
